### PR TITLE
pdksync - (IAC-973) - Update travis/appveyor to run on new default branch `main`

### PR DESCRIPTION
**Pull request by david22swan**
(IAC-973) - Update travis/appveyor to run on new default branch `main`
pdk version: `1.18.1`

### CODE REVIEW

**Comment by pmcmaw** [comment]
@david22swan there are quite a few code block removals here, are they intended?

**Comment by david22swan** [comment]
@pmcmaw All changes where done by the pdksync so they should be correct

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
```
Comment (sanfrancrisko) [inline .gitlab-ci.yml:1]: Should we be adding this file? I don't see it in any of our other supported modules.

Comment (david22swan) [reply .gitlab-ci.yml]: Removed

**Pull request merged**